docs/introduction/zns.md

```diff
@@ -31,13 +31,13 @@ at USENIX ATC 2021.
 
 ## Overview
 
-The ZNS SSD follows the Zoned Storage Model. This standards-based architecture,
+ZNS SSDs follows the Zoned Storage Model. This standards-based architecture,
```

Comment:
s/follows/follow/ - The verbs don't agree if it says "SSDs follows".

Reply:
Good catch, updated.